dashboard/lib/programming_expression_autocomplete.rb

```diff
@@ -0,0 +1,13 @@
+class ProgrammingExpressionAutocomplete < AutocompleteHelper
```

Comment:
I believe this should be in `dashboard/lib/queries`; see https://github.com/code-dot-org/code-dot-org/tree/staging/dashboard/lib/queries for an explanation of Query Objects

Reply:
`dashboard/lib` is where all the other autocomplete classes live, including the superclass. Not against moving them but I think that's out of the scope of this PR

Reply:
ah, fair. Let's move em all together, then, but in a follow-up PR

Reply:
https://codedotorg.atlassian.net/browse/PLAT-842